chain/client/src/test_utils.rs

```diff
@@ -1558,6 +1548,55 @@ impl TestEnv {
     pub fn get_runtime_config(&self, idx: usize, epoch_id: EpochId) -> RuntimeConfig {
         self.clients[idx].runtime_adapter.get_protocol_config(&epoch_id).unwrap().runtime_config
     }
+
+    /// Create and process a SIR tx from actions and return the execution outcome.
+    ///
+    /// The signer defines both sender and receiver of the transaction, hence it
+    /// is a Sender-is-Receiver transaction.
+    pub fn execute_sir_tx(
+        &mut self,
+        actions: Vec<Action>,
+        signer: &InMemorySigner,
+    ) -> FinalExecutionOutcomeView {
+        let tip = self.clients[0].chain.head().unwrap();
+        let tx = Transaction {
+            signer_id: signer.account_id.clone(),
+            receiver_id: signer.account_id.clone(),
+            public_key: signer.public_key(),
+            actions,
+            nonce: tip.height + 1,
+            block_hash: tip.last_block_hash,
+        };
+        let signed_tx = tx.sign(signer);
+        let tx_hash = signed_tx.get_hash().clone();
+        self.clients[0].process_tx(signed_tx, false, false);
```

Comment:
Magically defaulting to `0` client is somewhat odd, but I also don't feel like we should add client_id param everywhere. I'd say this OK, maybe at some point in the future we'll learn how a better API would look like.